src/client/bb-client.js

```diff
@@ -284,6 +284,11 @@ BBClient.ready = function(input, callback, errback){
 
   if (isFakeOAuthToken()) {
     accessTokenResolver = completePageReload();
+    if (window.history.replaceState
```

Comment:
Thanks @zplata! Can you add a comment explaining this block (and what happens in browsers that don't support `replaceState`)?

Reply:
@jmandel Addressed in 2nd commit